src/get-started/learn-more.md

```diff
@@ -1,54 +1,163 @@
+
+The following resources cover a few specific layout concepts that are very common, and a bit more involved than using rows and columns.
+
+#### Scrollable widgets
+
+There are several Flutter widgets that facilitate scrolling through content in
+your application. For now, you’ll see how to use the most common widget for
+making any given page scrollable, as well as a widget that creates scrollable
+lists. You’ll also be introduced to a common pattern in Flutter - the builder
+pattern. The builder pattern is used in many Flutter widgets, including some
+scrolling widgets.
+
+1. Documentation: [Basic scrolling][]
+2. Video: [Builder - Flutter Widget of the Week][]
+3. Video: [ListView - Flutter Widget of the Week][]
+4. Example code: [Work with long lists][]
+5. Example code: [Create a horizontal list][]
+6. Example code: [Create a grid list][]
+7. Video: [PageView - Flutter Widget of the Week][]
+
+#### Stacks
+
+Stacks give you more flexibility in placing widgets in Flutter by allowing you
+to specify exact locations of widgets within their parent, including on top of
+each other. If you’re coming from the web, `Stacks` solve the same problem
+as `z-index`.
+
+If you followed both of the tutorials mentioned on this page, you’ve already
+worked with Stacks. If not, these resources provide an overview.
+
+1. Video: [Stack - Flutter Widget of the Week][]
+2. Documentation: [Stack documentation][]
+
+#### Overlays
+
+Overlays are used when you have widgets that are logically related to each
+other, but visually separate, such as tooltips.
+
+1. Video: [OverlayPortal - Flutter Widget of the Week][]
+
+### Adaptive Layouts
+
+Because Flutter is used to create mobile, tablet, desktop and web apps, it’s
+likely you’ll need to adjust your application to behave differently depending on
+things like screen-size, input types, and more. The following resources start by
+showing the most important widgets when building adaptive layouts, and finish
+with a Codelab that has you build an adaptive layout yourself.
+
+1. Video: [LayoutBuilder - Flutter Widget of the Week][]
+2. Video: [MediaQuery - Flutter Widget of the Week][]
+3. Tutorial: [Adaptive Apps codelab][]
+4. Video: [Building platform adaptive apps][]
+
+### DevTools and Debugging for Layout
+
+Flutter has a robust suite of DevTools that will help you work with any number
+of aspects of Flutter development. The “Widget Inspector” tool is particularly
+useful when building Layouts (and working with widgets in general).
+
+Additionally, perhaps the most common error you’ll run into while building a
+Flutter application is due to incorrectly using Layout widgets, and is referred
+to as the “unbounded constraints” error. If there was only one type error you
+should be prepared to confront before you start building Flutter apps, it’d be
+this type.
+
+1. Article: [DevTools - Widget Inspector][]
+2. Video: [Unbounded height and width - Decoding Flutter][]
+
+### Advanced Content
+
+- [2D Scrolling][]
```

Comment:
Question: would we expect (as a general pattern) to have at least a sentence or two under every `###` heading introducing whatever items are there?

Reply:
I envisioned (and briefly talked to Khanh about) having an Advanced Content and Reference section at the end of topics, both of which have optional content.  I'm not sure what I'd write under these headings, but I'm open to ideas. 

On the current docs.flutter.dev/ui/layout page, there's a "Other resources" section with the description: "The following resources might help when writing layout code." I'm going to start by copying that.

Reply:
I'm also happy to just skip the "advanced content" altogether